polytracker/src/passes/bb_splitting_pass.cpp

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
I think we can revert this, there was a one line change that I originally needed for M1 compatibility and I think this file is just no longer relevant. I can make the change and test if you want.

Reply:
sure, go for it! thank you

Reply:
Done. I couldn't find the PR that remove that file though, I thought it must have been the sections PR but I don't see it. 🤔 Probably best to wait for CI before a merge.